packit/api.py

```diff
@@ -383,6 +384,11 @@ def create_srpm(self, output_file: str = None, upstream_ref: str = None) -> Path
         return srpm_path
 
     def status(self):
+        self.up.local_project.git_project = GithubProject(
```

Comment:
This should be taken care of inside LocalProject, not here. We should really investigate how come the git_project is None and address the root cause.

Reply:
I think it could be also here in this method: https://github.com/packit-service/packit/blob/9e16033e1223a74b3f98d24b239d934bcd05f12f/packit/upstream.py#L69

Reply:
in this case, LocalProject of the Upstream class is set in here: https://github.com/packit-service/packit/blob/master/packit/cli/types.py#L58